.github/workflows/prod-deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Production Deploy
+
+on:
+  release:
+    types:
+      - published
```

Comment:
Should this be on - published or on - released?  Seems like there are some small wrinkles as defined here 
https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=released#release

Reply:
`released`
> A release was published, or a pre-release was changed to a release.

`publish`
> A release, pre-release, or draft of a release was published.

Hmmm, it's a little hard to piece together.  Do you think we're missing the part when a pre-release is promoted to a release?  But I'm also not sure what we'd use a pre-release for in the first place.

Reply:
I agree it's not the clearest difference.  The thing that raised my brow was a draft of a release being published, would that mean we would deploy a draft release when it's created in the UI?

Reply:
Ah, I did look into that.  When a draft of a release happens, then only `created` happens.  When the draft goes out of draft, it is `published`.